bazel-mode.el

```diff
@@ -107,6 +107,10 @@
   (add-hook 'before-save-hook #'bazel-mode--buildifier-before-save-hook
             nil :local))
 
+;;;###autoload
+(add-to-list 'auto-mode-alist
+             (cons (rx (or "BUILD" "WORKSPACE" ".bazel" ".bzl" ".BUILD") eos) #'bazel-mode))
```

Comment:
You should probably start these with `/` so that files like `foobarBUILD` aren't matched.

Reply:
The prefix "/" works like "^BUILD$" but I can't find document about the usage.
Could you provide some manual about this trick?

Reply:
The keys in `auto-mode-alist` are matched against the entire expanded filename. Starting a match with `/` just matches the directory separator. The manual entry for `auto-mode-alist` has a few examples for this.

Reply:
Oh, it makes sense. Thanks :)